aip/0163.md

```diff
@@ -0,0 +1,43 @@
+---
+aip:
+  id: 163
+  state: reviewing
+  created: 2019-12-16
+permalink: /163
+redirect_from:
+  - /0163
+---
+
+# Validating changes
+
+Occasionally, a user wants to validate an intended change to see what the
+result will be before actually making the change. For example, a request to
+provision new servers in a fleet will have an impact on the overall fleet size
+and cost, and could potentially have unexpected downstream effects.
+
+## Guidance
+
+APIs **may** provide an option to validate, but not actually execute, a
+request, and provide the output that it would have provided if the request was
+actually executed.
```

Comment:
For create operations, that output would normally include something with a resource name - I'm not sure it's a good idea to populate that if the resource hasn't actually been created.

Reply:
Concur; will amend.

Reply:
Added.